tools/test/test_upload_test_stats.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import os
+
+IN_CI = os.environ.get("CI")
```

Comment:
very nit but for consistency, I've noticed you naming these IS_CI in the IN_CI removal PR.
```suggestion
IS_CI = os.environ.get("CI")
```

Reply:
I'll do the rename in that PR